src/Sylius/Bundle/ShippingBundle/DateAssigner/ShipmentShippedAtDateAssignerInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ShippingBundle\DateAssigner;
+
+use Sylius\Component\Shipping\Model\ShipmentInterface;
+
+interface ShipmentShippedAtDateAssignerInterface
```

Comment:
What about `ShippingDateAssignerInterface` in a namespace `Assigner`? `ShippedAt` is a technical detail (the name od the property), `Shipment` is obvious, as we get a `ShipmentInterface` argument for the only function